build_runner/README.md

```diff
@@ -83,22 +151,29 @@ from the current package.
 In general it is best to be as specific as possible with your `InputSet`s,
 because all matching files will be provided to `declareOutputs`.
 
-### Outputs
+> Future versions of `build_runner` may aid in automatically generating
+> `InputSet`s based on the builders being used! See [issue #353][issue_353].
 
-You may only output files in the current package, but anywhere in the current
-package is allowed.
-
-You are not allowed to overwrite existing files, only create new ones.
+### Outputs
 
-Outputs from previous builds will not be treated as inputs to later ones.
+* You may output files anywhere in the current package.
+* You are not allowed to overwrite existing files, only create new ones.
+* Outputs from previous builds will not be treated as inputs to later ones.
+* You may use a previous `BuildAction`'s outputs as an input to a later action.
 
 ### Source control
 
 This package creates a top level `.dart_tool` folder in your package, which
-should not be submitted to your source control repo (likely this just means
-adding '.dart_tool' to your '.gitignore' file).
+should not be submitted to your source control repository. You can see [our own
```

Comment:
I was thinking that we could _potentially_ speed up travis by allowing you to actually submit your asset graph and generated assets, but for now we should keep this wording I think :). We would have to do invalidation based on content hash instead of last modified time at the very least to enable that, and it still might be a bad idea.

Reply:
Definitely worth thinking about (take advantage of incremental builds), but let's revisit later.